sdk/keyvault/azure-keyvault-securitydomain/README.md

```diff
@@ -0,0 +1,80 @@
+
+
+# Azure Keyvault Securitydomain client library for Python
+<!-- write necessary description of service -->
+
+## Getting started
+
+### Install the package
+
+```bash
+python -m pip install azure-keyvault-securitydomain
+```
+
+#### Prequisites
+
+- Python 3.8 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Azure Keyvault Securitydomain instance.
+#### Create with an Azure Active Directory Credential
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
+
+Use the returned token credential to authenticate the client:
+
+```python
+>>> from azure.keyvault.securitydomain import KeyVaultClient
```

Comment:
`SecurityDomainClient` ?

Reply:
The readme hasn't been updated yet; I'm preparing a commit with that update and samples at the moment